free-2.md

```diff
@@ -58,12 +123,34 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I realize that the language design aspect of this project would probably be minimal,
+and it's an indea that I would need to reconsider or specifiy better to act like a
+DSL. The project is highly dependent on the input, and it gives the user perhaps too
+much flexibility in determining what inputs they want to use. So there would definitely
+need to be more structure and a sort of paradigm that dictates what kinds of search
+parameters and inputs the user can choose from. The "systems" aspect would take up the
+bulk of the time, because it would possibly require some sort of algorithm design and
+text parsing in order to identify songs. It would also require significant knowledge
+of how to use different APIs that contain song or lyric databases.
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+Reflecting on this idea, I realize that it was easy to imagine it but seems very
+impractical to execute. After thinking about this idea for a bit, I realize that
+it is too ambitious because of all of the data I would need access to and a knowledge 
```

Comment:
I would agree. There is a lot that would go into the systems side, and I think designing the interface would be rather straightforward